assets/javascripts/modules/analytics/ga.js

```diff
@@ -16,6 +18,10 @@ define(['utils/cookie',
         })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
         /*eslint-enable */
 
+        // Do JellyFish first as they don't want our custom dimensions
+        ga('create', 'UA-44575989-1', 'auto', 'jellyfishGA');
```

Comment:
That's cool.
I didn't explain myself too well. Currently our default tracker doesn't have a name - we could it something like membershipPropertyTracker
We then set each of the custom dimensions with the relevant tracker. So in this case we would have something like:
ga('membershipPropertyTracker.set', 'dimension5', 'subscriptions');
etc...
We can then be more explicit with what goes to what tracker without having to worry about having the Jellyfish pageview send go first.
This is what is being done on Frontend.
Does that make sense?